### PR TITLE
Board styling

### PR DESCRIPTION
**Pull request by Phavion**
style board pages, replace hrefs with Link, add mobile version

### CODE REVIEW

**Comment by emielvanrijn** [comment]
Looking quite good already! Just some suggestions:
1. We could make the buttons for prev. and next board colored as the boardcolor of the board currently viewed.
2. I think pulling the buttons outside of the card and putting them in a flex-container is a good idea.
3. Maybe we should put the 'o.d.z.' in the card, so only the title "Bestuur 12" is above the card. This is more in line with the rest of the website styling

File: src/pages/vereniging/besturen.jsx [hunk 1]
```diff
@@ -13,57 +13,37 @@ const BoardPage = props => {
       <h2>{page.title}</h2>
       <Markdown>{page.content.content}</Markdown>
       <h3>Het huidig bestuur</h3>
-      <CurrentBoard>{getCurrentBoard(boards)}</CurrentBoard>
+      <Grid stretched stackable doubling columns={3}>
```
Comment (cdfa) [inline src/pages/vereniging/besturen.jsx]: Why did you use a `<Grid>` here? There's only 1 current board right?

Comment (cdfa) [inline src/pages/vereniging/besturen.jsx]: I think it would be nice to have the current board card a little larger and centered.

Comment (emielvanrijn) [reply src/pages/vereniging/besturen.jsx]: Yes I agree!

Comment (emielvanrijn) [reply src/pages/vereniging/besturen.jsx]: And maybe also their names already displayed?

Comment (Phavion) [reply src/pages/vereniging/besturen.jsx]: Sure, but what is the extra effect of having a board page for them then

Comment (emielvanrijn) [reply src/pages/vereniging/besturen.jsx]: Just consistency with the other boards. But I also like the way it is now :).

**Comment by Phavion** [comment]
Can't approve it but still: LGTM :P

**Pull request merged**